File: C#/FizzBuzz.cs
// Simple version of FizzBuzz in bash
// Author: @jmmille

using System;

namespace FizzBuzz
{
    class Program
    {
        static void Main(string[] args)
        {
            for (int i = 0; i < 101; i++)
            {
                if (i % 3 == 0 && i % 5 == 0)
                {
                    Console.WriteLine("fizzbuzz");
                    continue;
                } else if (i % 3 == 0){
                    Console.WriteLine("fizz");
                    continue;
                } else if (i % 5 == 0)
                {
                    Console.WriteLine("buzz");
                    continue;
                }
                Console.WriteLine(i);
            } 
        }
    }
}

File: Kotlin/fizzBuzz.kt

// A simple Kotlin Program for fizzbuzz
// Author: @rohfl

import java.util.Scanner  
fun main(args: Array<String>) {
    val read = Scanner(System.`in`)  
    var number = read.nextInt()  
    for (i in 1..number) {
        val result: String =
            if (i % 15 == 0)
				"FizzBuzz"
            else if (i % 3 == 0)
				"Fizz"
            else if (i % 5 == 0)
				"Buzz"
            else
				"${i}"
        println(result)
    }
}